scripts/deploy

```diff
@@ -1,16 +0,0 @@
-#!/bin/bash
-set -e
-
-source ${SCRIPTS_DIR}/lib/debug_functions
-source ${SCRIPTS_DIR}/lib/version
-
-function load_image() {
-    docker pull ${1}:latest
-    docker tag ${1}:latest ${1}:$VERSION
-}
-
-load_image quay.io/submariner/submariner
-load_image quay.io/submariner/submariner-route-agent
-load_image quay.io/submariner/submariner-globalnet
```

Comment:
You removed this part but it's necessary since shipyard doesn't build these images, that's why the CI fails on this patch.
I'd keep this script or alternatively add this as a "pre_deploy" target which the deploy needs (since you want to share the deploy recipe)

Reply:
I fixed `deploy.sh` to pull images it can’t find locally instead.